tests/db_functions/tests.py

```diff
@@ -25,6 +25,83 @@ def truncate_microseconds(value):
 
 class FunctionTests(TestCase):
 
+    def test_strindex(self):
+        Author.objects.create(name='George. R. R. Martin')
+        Author.objects.create(name='J. R. R. Tolkien')
+        Author.objects.create(name='Terry Pratchett')
+        authors = Author.objects.annotate(fullstop=StrIndex('name', 'R.'))
+        self.assertQuerysetEqual(
+            authors.order_by('name'),
+            [9, 4, 0],
+            lambda a: a.fullstop
+        )
+
+    def test_strindex_textfield(self):
+        Article.objects.create(title="How to Django", text=lorem_ipsum, written=timezone.now())
+        Article.objects.create(title="How to Tango", text='Won\'t find anything here.', written=timezone.now())
+
```

Comment:
Please be consistent with blank lines. I think none are needed for all these new tests since they're fairly short.